ui/raidboss/data/06-ew/dungeon/lapis_manalis.txt

```diff
@@ -0,0 +1,283 @@
+99.9 "Call of the Mountain" sync / 1[56]:[^:]*:Albion:7A7C:/
+105.3 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+110.9 "Wildlife Crossing 1" sync / 00:0839::A rumbling in the distance draws closer.../ window 30,30 duration 15
+112.1 "Left/Right Slam" sync / 1[56]:[^:]*:Albion:(802E|802D):/
+119.6 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+124.9 "Wildlife Crossing 2" sync / 00:0839::More rumbling can be heard in the distance.../ window 30,30 duration 15
+125.9 "Left/Right Slam" sync / 1[56]:[^:]*:Albion:(802E|802D):/
+135.1 "Albion's Embrace" sync / 1[56]:[^:]*:Albion:7A85:/
+143.1 "Knock on Ice" sync / 1[56]:[^:]*:Albion:7A7E:/
+145.1 "Knock on Ice" sync / 1[56]:[^:]*:Albion:7A7F:/
+151.2 "Icebreaker" sync / 1[56]:[^:]*:Albion:7A81:/
+154.2 "Icy Throes" sync / 1[56]:[^:]*:Albion:7A82:/
+160.4 "Icy Throes" sync / 1[56]:[^:]*:Albion:(800F|7A83):/
+160.8 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+165.4 "Icy Throes" sync / 1[56]:[^:]*:Albion:7FB9:/
+169.4 "Roar of Albion" sync / 1[56]:[^:]*:Albion:7A84:/ window 30,30 jump 86.8
+182.6 "Call of the Mountain" sync / 1[56]:[^:]*:Albion:7A7C:/
+188.0 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+193.4 "Wildlife Crossing 1" sync / 00:0839::A rumbling in the distance draws closer.../ window 30,30 duration 15
+194.6 "Left/Right Slam" sync / 1[56]:[^:]*:Albion:(802E|802D):/
+202.2 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+207.2 "Wildlife Crossing 2" sync / 00:0839::More rumbling can be heard in the distance.../ window 30,30 duration 15
+208.3 "Left/Right Slam" sync / 1[56]:[^:]*:Albion:(802E|802D):/
+217.5 "Albion's Embrace" sync / 1[56]:[^:]*:Albion:7A85:/
+225.5 "Knock on Ice" sync / 1[56]:[^:]*:Albion:7A7E:/
+227.5 "Knock on Ice" sync / 1[56]:[^:]*:Albion:7A7F:/
+233.8 "Icebreaker" sync / 1[56]:[^:]*:Albion:7A81:/
+236.8 "Icy Throes" sync / 1[56]:[^:]*:Albion:7A82:/
+243.0 "Icy Throes" sync / 1[56]:[^:]*:Albion:(800F|7A83):/
+243.4 "--sync--" sync / 1[56]:[^:]*:Albion:802C:/
+248.1 "Icy Throes" sync / 1[56]:[^:]*:Albion:7FB9:/
+252.1 "Roar of Albion" sync / 1[56]:[^:]*:Albion:7A84:/
+265.3 "Call of the Mountain" sync / 1[56]:[^:]*:Albion:7A7C:/
+
+#~~~~~~~~~~~~~~~#
+# Galatea Magna #
+#~~~~~~~~~~~~~~~#
+
+1000.0 "--sync--" sync / 00:0839::The Forum Messorum will be sealed off/ window 1000,1
+1010.2 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7F71:/
+1015.2 "Waxing/Waning Cycle" sync / 1[56]:[^:]*:Galatea Magna:(7A91|7F6E):/
+1016.8 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7F6F:/
+1017.2 "Waxing/Waning Cycle" sync / 1[56]:[^:]*:Galatea Magna:(7A93|7F70):/
+1021.2 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7F71:/
+1028.3 "Soul Scythe" sync / 1[56]:[^:]*:Galatea Magna:7A9A:/
+1031.0 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7F71:/
+1037.4 "Soul Nebula" sync / 1[56]:[^:]*:Galatea Magna:7A9E:/
+1049.1 "Scarecrow Chase" sync / 1[56]:[^:]*:Galatea Magna:7A9B:/
+1049.3 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7F71:/
+1050.3 "--sync--" sync / 1[56]:[^:]*:Galatea Magna:7A9D:/
```

Comment:
Are either of these autos? There's a lot of these syncs.  Should they just be ignored since there's plenty of other abilities?

Reply:
`7F71` is unknown; probably boss re-centering, since it always targets the environment with it (she re-centers a ton in that fight).  Her autos are `366`.  `7A9D` is part of Scarecrow Chase; it's the actual teleports before the cross-attacks.  I'll ignore these.